src/Reason.ml

```diff
@@ -30,12 +30,12 @@ module Binary_reason = struct
      (src/migrate_parsetree_ast_io.ml) *)
   type ast =
     | Impl :
-        (module Migrate_parsetree_versions.OCaml_version
+        (module Migrate_parsetree.Versions.OCaml_version
```

Comment:
We're waiting for a release of migrate-parsetree to push that on master (https://github.com/ocaml-ppx/ocaml-migrate-parsetree/issues/67)